packages/demo/backend/src/router.ts

```diff
@@ -50,18 +58,28 @@ router.get(
   walletController.getBalance,
 )
 router.post('/wallet/:userId/fund', authMiddleware, walletController.fundWallet)
-router.post('/wallet/send', walletController.sendTokens)
 
 // Lend endpoints
 router.get('/lend/markets', lendController.getMarkets)
+router.get('/lend/marketsV1', lendController.getMarketsV1)
 router.get('/lend/market/:chainId/:marketAddress', lendController.getMarket)
 router.get(
   '/lend/market/:chainId/:marketAddress/position/:walletId',
   lendController.getPosition,
 )
 router.post('/lend/position/open', authMiddleware, lendController.openPosition)
+router.post(
+  '/lend/position/openV1',
+  authMiddleware,
+  lendController.openPositionV1,
+)
 router.post(
   '/lend/position/close',
   authMiddleware,
   lendController.closePosition,
 )
+router.post(
+  '/lend/position/closeV1',
```

Comment:
it feels more common to see the version first, like /v1/lend/..., mind updating all these?